pkg/cli/democluster/demo_cluster.go

```diff
@@ -2042,11 +2053,40 @@ func (c *transientCluster) ExpandShortDemoURLs(s string) string {
 	if !strings.Contains(s, "demo://") {
 		return s
 	}
-	u, err := c.getNetworkURLForServer(context.Background(), 0, false, forSystemTenant)
-	if err != nil {
-		return s
+
+	for _, match := range regexp.MustCompile(`demo://[a-zA-Z0-9:@/]+`).FindAllString(s, -1) {
```

Comment:
For the character class here, are we getting much out of being strict here. Could we accept all non-whitespace?